### PR TITLE
input files for cloud cost with surface wind

### PR DESCRIPTION
**Pull request by stemiglio**
## Description

This change adds the input files for testing the calculation of the cloud cost used for quality control when surface winds are also included in the state vector. 

## Dependencies

This PR is required by JCSDA-internal/ufo/pull/2851

### CODE REVIEW

**Comment by DJDavies2** [comment]
I run the ufo ctests with the ufo branch and this ufo-data branch and still got a failure:

ufo_test_tier1_test_ufo_function_cloudcost_surfwind:

 2023/06/05  14:38:57  fatal error in module /home/d03/frwd/cylc-run/ufo-2851/share/mo-bundle/rttov/src/rttov/coef_io/rttov_coef_io_mod.F90:0163
     Cannot find any coefficient file named: Data//rtcoef_gpm_1_gmi.dat/.bin/.h5/.H5
 2023/06/05  14:38:57  fatal error in module /home/d03/frwd/cylc-run/ufo-2851/share/mo-bundle/rttov/src/rttov/coef_io/rttov_read_coefs.F90:0238
     
fatal error reading coefficients: rtcoef_gpm_1_gmi
OOPS Ending   2023-06-05 14:38:57 (UTC+0000)
Rank 0 [Mon Jun  5 14:38:57 2023] [c2-0c2s11n1] application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0

**Review by smnewman — APPROVED** [review]
Thanks @stemiglio, I can see the test files contain 11 locations with observations comprising 4 channels (10-13). Assuming the ctest passes with 1.0e-5 tolerance this looks good.

**Comment by smnewman** [comment]
> I run the ufo ctests with the ufo branch and this ufo-data branch and still got a failure:
>
> ufo_test_tier1_test_ufo_function_cloudcost_surfwind:
>
> 2023/06/05 14:38:57 fatal error in module /home/d03/frwd/cylc-run/ufo-2851/share/mo-bundle/rttov/src/rttov/coef_io/rttov_coef_io_mod.F90:0163
> Cannot find any coefficient file named: Data//rtcoef_gpm_1_gmi.dat/.bin/.h5/.H5
2023/06/05 14:38:57 fatal error in module /home/d03/frwd/cylc-run/ufo-2851/share/mo-bundle/rttov/src/rttov/coef_io/rttov_read_coefs.F90:0238
>
> fatal error reading coefficients: rtcoef_gpm_1_gmi
OOPS Ending 2023-06-05 14:38:57 (UTC+0000)
Rank 0 [Mon Jun 5 14:38:57 2023] [c2-0c2s11n1] application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0

@david-rundle do the RTTOV ctests rely on coefficients in https://github.com/JCSDA-internal/rttov/tree/develop/rtcoef_rttov12/rttov7pred54L? (If so, then we would need to add `rtcoef_gpm_1_gmi.dat`?)

**Comment by stemiglio** [comment]
See https://github.com/JCSDA-internal/rttov/pull/32 where `rtcoef_gpm_1_gmi.dat` is added

**Review by DJDavies2 — APPROVED** [review]

**Review by fcvdb — APPROVED** [review]
There are some warnings when reading file:
 `Data/ufo/testinput_tier_1/jopa_gmihigh_20210701T1200Z_out_m.nc4` 
in test:
 `ufo_test_tier1_test_ufo_function_cloudcost_surfwind `

```
149: WARNING: ioda::Variable: hdf and netcdf fill value specifications do not match.
149:     hdf fill value property: 9.96921e+36
149:     netcdf _FillValue attribute: -3.3688e+38
149: WARNING: selecting the netcdf _FillValue attribute value: -3.3688e+38
```

```
149:     hdf fill value property: -2147483647
149:     netcdf _FillValue attribute: -2147483643
149: WARNING: selecting the netcdf _FillValue attribute value: -2147483643
149: WARNING: ioda::Variable: hdf and netcdf fill value specifications do not match.
```

File: testinput_tier_1/jopa_gmihigh_20210701T1200Z_out_m.nc4
Comment (PatNichols) [inline testinput_tier_1/jopa_gmihigh_20210701T1200Z_out_m.nc4:1]: This file seems a little large...Is it possible to shrink it down some ? Thanks.

**Review by PatNichols — CHANGES_REQUESTED** [review]

File: testinput_tier_1/jopa_gmihigh_20210701T1200Z_out_m.nc4
Comment (stemiglio) [reply testinput_tier_1/jopa_gmihigh_20210701T1200Z_out_m.nc4:1]: I have just upload a smaller test file and checked that the test still pass

**Pull request closed without merging**